webhooks-extension/docs/Limitations.md

```diff
@@ -0,0 +1,57 @@
+# Limitations
+<br/>
+
+- Only GitHub webhooks are currently supported.
+- Only `push` and `pull_request` events are currently supported, these are the events defined on the webhook.
+- A limited number of pipelines will currently work with this implementation (see below)
```

Comment:
Wouldn't even mention see below here as it's centimetres below

Reply:
ok